boards/phytec/phyboard_lyra/Kconfig.phyboard_lyra

```diff
@@ -5,8 +5,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-config BOARD_PHYBOARD_LYRA_AM62X
-	select SOC_AM6234_A53 if BOARD_PHYBOARD_LYRA_AM62X_AM6234_A53
-	select SOC_AM6234_M4 if BOARD_PHYBOARD_LYRA_AM62X_AM6234_M4
+config BOARD_PHYBOARD_LYRA
+	select SOC_AM6234_A53 if BOARD_PHYBOARD_LYRA_AM6234_A53
+	select SOC_AM6234_M4 if BOARD_PHYBOARD_LYRA_AM6234_M4
```

Comment:
make changes in the first commit

Reply:
Commit title is `boards: phytec: Remove am62x from phyboard_lyra_am62x`. Why should I move this change into another commit?

Reply:
You're adding hwmv1 stuff which was outdated a month ago then correcting it in another commit, this is a flat out no

Reply:
This change just drops am62x. How is this related to hwmv1?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/70913/commits/477835129407fa80dcd50005209b61f295054213 this change, then in the next commit doing
```
- The Zephyr phyboard_lyra_am62x_am6234_a53 board configuration supports the following hardware
+ The Zephyr phyboard_lyra/am6234/a53 board configuration supports the following hardware
```
`phyboard_lyra_am62x_am6234_a53` is hwmv1, let's stick to hwmv2 throughout rather than having commits adding hwmv1 terminology

Reply:
isn't this thread about something different? Pushed a clean history btw.